Client/src/Features/UI/uiSlice.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+// Initial state for UI settings.
+// Add more settings as needed (e.g., theme preferences, user settings)
+const initialState = {
+  table: {
+    rowsPerPage: 5,
+  },
+  sidebar: {
+    collapsed: false,
+  },
+};
```

Comment:
Is there only ever going to be one table whose state we need to keep track of?

Maybe we should make this more extensible by having a state object?

```
{
   montiorsTable: { rowsPerPage: 5}
   otherTable: {rowsPerPage: 10}
}
```

What do you think?

Either that or separate slices for each?

Reply:
> Is there only ever going to be one table whose state we need to keep track of?

Not sure, it depends if we want to add the rows functionality to any other tables. 

>Maybe we should make this more extensible by having a state object?

Yea I agree.

>Either that or separate slices for each?

I don't think we need separate slices for each table, we won't be tracking too many variables. All I can think of right now is `rowsPerPage` and `sortOrder`.

Reply:
OK lets assume we'll have other tables to track in the future, seems more likely than not that it will come up.

Looks like that state object is the simplest way to go